nodes/src/nodes/ml_sklearn/code.py

```
@@ -0,0 +1,33 @@
+                return {'error': "Missing 'price'"}
+
+            try:
+                price = float(price)
+            except (ValueError, TypeError):
+                return {'error': 'Price must be a number'}
+
+            prediction = self.model.predict([[price]])
+
+            return {'prediction': float(prediction[0])}
+
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)